lib/arrow_web/components/limit_section.ex

```diff
@@ -359,6 +365,21 @@ defmodule ArrowWeb.LimitSection do
     Map.get(icon_paths, kind)
   end
 
+  defp get_limit_date_range_warning(start_date, end_date)
+       when start_date in [nil, ""] or end_date in ["", nil] do
+    ""
+  end
+
+  defp get_limit_date_range_warning(start_date, end_date) do
+    today = Date.utc_today()
```

Comment:
Are `start_date` and `end_date` in UTC timezone?

If they're local you will want to get `today` in local time as well. Otherwise, this could produce the wrong result if Arrow is used after ~6pm Eastern.
```suggestion
    today =
      DateTime.utc_now()
      |> DateTime.shift_zone!("America/New_York")
      |> DateTime.to_date()
```

Reply:
Good call, will need to shift.